test/functional/core/startup_spec.lua

```diff
@@ -294,24 +285,12 @@ describe('startup', function()
         os.remove('Xtest_shada')
       end)
 
-      assert_l_out(
-        'updatecount=0 shadafile=NONE loadplugins=false scripts=1\n',
-        nil,
-        nil,
-        '-',
-        script
-      )
+      assert_l_out('updatecount=0 shadafile=NONE loadplugins=false scripts=1\n', nil, nil, '-', script)
 
       -- User can override.
-      assert_l_out(
-        function(out)
-          return matches('updatecount=99 shadafile=Xtest_shada loadplugins=true scripts=2%d\n', out)
-        end,
-        { '+set updatecount=99', '-i', 'Xtest_shada', '+set loadplugins', '-u', 'NORC' },
-        nil,
-        '-',
-        script
-      )
+      assert_l_out(function(out)
+        return matches('updatecount=99 shadafile=Xtest_shada loadplugins=true scripts=2%d\n', out)
+      end, { '+set updatecount=99', '-i', 'Xtest_shada', '+set loadplugins', '-u', 'NORC' }, nil, '-', script)
```

Comment:
this case is an example where 120 seems too long.

but i'm ok with it as long as we have auto-formatting for all files somehow or other.